src/main/java/org/jabref/logic/importer/util/FileFieldParser.java

```diff
@@ -18,10 +18,15 @@ public class FileFieldParser {
     private final String value;
 
     private StringBuilder charactersOfCurrentElement;
+
     private boolean windowsPath;
 
     public FileFieldParser(String value) {
-        this.value = value;
+        if (value == null) {
+            this.value = null;
+        } else {
+            this.value = value.replace("$\\backslash$", "\\");
```

Comment:
Why not call the latex 2 Unicode formatter to capture more cases

Reply:
I would need to rewrite the logic of the latextounicode formatter:

![grafik](https://github.com/JabRef/jabref/assets/1366654/bad32fb5-c590-4d8d-b23c-8cb40a6a3376)

Expected :[ParsedFileField{description='', link='C:/Users/XXXXXX/AppData/Local/Mendeley Ltd./Mendeley Desktop/Downloaded/Brown - 2017 - Physical test methods for elastomers.pdf', fileType='pdf'}]
Actual   :[ParsedFileField{description='', link='Cbackslash{}{', fileType='}{/}Users/XXXXXX/AppData/Local/Mendeley Ltd./Mendeley Desktop/Downloaded/Brown - 2017 - Physical test methods for elastomers.pdf'}]

Expected :[ParsedFileField{description='test:;', link='wei2005ahp.pdf', fileType='PDF'}]
Actual   :[ParsedFileField{description='test  ', link='wei2005ahp.pdf', fileType='PDF'}]


Should I commit nevertheless?

Reply:
I would go with the simple solution first and leave this whooe latex removing for later

Reply:
One would probaly treat each path segment separately by the formatter

Reply:
In the original example, there were no umlauts there. I was thinking: better a quick fix than some effort on catching the special cases 😇